examples/EJS/nodejs/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "current-ejs",
+  "version": "0.0.1",
+  "description": "A simple EJS-expanding server.",
+  "homepage": "https://github.com/C5T/Current",
+  "repository": {
+    "url": "git@github.com:C5T/Current.git",
+    "type": "git"
+  },
+  "bugs": {
+    "url": "https://github.com/C5T/Current/issues"
+  },
+  "author": {
+    "email": "dmitry.korolev@gmail.com",
+    "url": "http://dimakorolev.com/",
+    "name": "Dima Korolev"
+  },
+  "licenses": [{
+    "type": "MIT"
+  }],
+  "license": "MIT",
+  "tags": [
+    "current",
+    "ejs",
+    "cpp",
+    "backend"
+  ],
+  "keywords": [
+    "current",
+    "ejs",
+    "cpp",
+    "backend"
+  ],
+  "main": "server.js",
+  "dependencies": {
+    "body-parser": ">=1.15.2",
```

Comment:
Why `>=`? Either nothing (exact version), or `^` (latest with the same major version).